src/test/java/roomescape/integration/ThemeIntegrationTest.java

```diff
@@ -0,0 +1,116 @@
+package roomescape.integration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import io.restassured.RestAssured;
+import io.restassured.common.mapper.TypeRef;
+import io.restassured.http.ContentType;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+class ThemeIntegrationTest extends IntegrationTest {
+    @Nested
+    @DisplayName("테마 목록 조회 API")
+    class FindAllTheme {
+        @Test
+        void 테마_목록을_조회할_수_있다() {
+            RestAssured.given().log().all()
+                    .when().get("/themes")
+                    .then().log().all()
+                    .statusCode(200)
+                    .body("size()", is(1));
+        }
+    }
+
+    @Nested
+    @DisplayName("인기 테마 목록 조회 API")
+    class FindAllPopularTheme {
+        @Test
+        void 최근_일주일동안_예약_건수_많은_순서대로_10개_테마를_인기_테마로_조회할_수_있다() {
+            jdbcTemplate.update("INSERT INTO reservation (name, date, time_id, theme_id) VALUES (?, ?, ?, ?)",
+                    "브라운", "1999-09-18", "1", "1");
+
+            RestAssured.given().log().all()
+                    .when().get("/themes/ranking")
+                    .then().log().all()
+                    .statusCode(200)
+                    .body("size()", is(1));
+
+            List<Map<String, Object>> response = RestAssured.get("/themes/ranking")
+                    .as(new TypeRef<>() {
+                    });
+            assertThat(response.get(0)).containsEntry("id", 1);
```

Comment:
TypeRef는 이렇게 표현하는 경우가 많아요.
```suggestion
.as(new TypeRef<>() {});
```